mscore/CMakeLists.txt

```diff
@@ -644,6 +644,12 @@ if (MINGW)
       install( FILES
          ${MINGW_ROOT}/bin/libgcc_s_seh-1.dll
          DESTINATION bin)
+      if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.12.4")
```

Comment:
I don't see `Qt5Widgets_VERSION` variable. Is it defined somewhere?

Reply:
Ah, I see https://doc.qt.io/archives/qt-5.5/cmake-manual.html#qt5widgets-macros

Reply:
Yep, that's where I got it from